test/unittests/test_analyse_xml_data/test_construct_and_analyze_graph.py

```diff
@@ -384,8 +384,7 @@ def test_remove_edges_to_stash_phantom_xml() -> None:
         Edge(2, 5, c_bdf_xml),
     ]
     graph.add_edges_from([e.as_tuple() for e in edges])
-    res_links_to_stash = _remove_edges_to_stash(graph, edges_to_remove, edges, remaining_nodes)
-    assert res_links_to_stash == [c_bdf_xml]
+    _remove_edges_to_stash(graph, edges_to_remove, edges, remaining_nodes)
```

Comment:
also here, the previous implementation was more explicit and probably easier to debug